src/config/dynamic-theme-fixes.config

```
@@ -6764,6 +6764,8 @@ leetcode.com
 leetcode-cn.com
 
 INVERT
+.node-text__Cnsr
+.logo__3xn0
```

```suggestion
img[alt="logo"]
```
Should be a little bit more clear what kind of selector it is.